dask_cuda/cuda_worker.py

```diff
@@ -152,6 +153,9 @@ def del_pid_file():
                 )
             if rmm_pool_size is not None:
                 rmm_pool_size = parse_bytes(rmm_pool_size)
+                if rmm_maximum_pool_size is not None:
+                    rmm_maximum_pool_size = parse_bytes(rmm_maximum_pool_size)
```

Comment:
Wondering if we should explicitly raise a warning if `rmm_maximum_pool_size` is specified without `rmm_pool_size`, to further clarify its behavior?

Reply:
Something like:

```suggestion
                
            if rmm_maximum_pool_size is not None:
                if rmm_pool_size is None:
                    warnings.warn(
                        "`rmm_maximum_pool_size` was specified without specifying "
                        "`rmm_pool_size` and will be ignored"
                    )
                rmm_maximum_pool_size = parse_bytes(rmm_maximum_pool_size)
```

Reply:
Yeah, that's a good idea. Maybe we can raise `ValueError` in those cases and update docstrings to reflect that @VibhuJawa ?

Reply:
Yeah. That looks good to me. Let me make that change and reflect it in the docstrings.

Reply:
Resolved with commit : 
https://github.com/rapidsai/dask-cuda/pull/827/commits/53e8572056c3719540a29bf148cb13bdf5867124

I decided to raise a ValueError inside the RMMSetup Class (see [link](https://github.com/rapidsai/dask-cuda/blob/53e8572056c3719540a29bf148cb13bdf5867124/dask_cuda/utils.py#L56-L60)) so that it gets picked up for both CLI and LocalCudaCluster.